metron-streaming/Metron-Common/src/test/java/org/apache/metron/enrichment/EnrichmentConfigTest.java

```diff
@@ -0,0 +1,253 @@
+package org.apache.metron.enrichment;
+
+import org.apache.metron.Constants;
+import org.apache.metron.domain.SourceConfig;
+import org.apache.metron.utils.JSONUtils;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class EnrichmentConfigTest {
+  @Test
+  public void testThreatIntel() throws Exception {
+    /*
+    {
+      "index": "bro",
```

Comment:
Ok, fair enough, I'm adding the multiline string utility for the unit tests and moving the config JSON to there, so we don't have to maintain them in 2 different places.  This is a similar pattern as they use in DataFu for their unit tests.